apps/docs/src/app/core/component-docs/select/examples/select-programmatic-example/select-programmatic-example.component.ts

```diff
@@ -8,17 +8,40 @@ import { Component } from '@angular/core';
 export class SelectProgrammaticExampleComponent {
 
     options: string[] = ['Apple', 'Pineapple', 'Tomato', 'Strawberry'];
+    selectedPosition = 0;
 
     selectedValue: string = this.options[0];
+    newValue = this.options[0];
 
     changeValue(): void {
         this.selectedValue = this.getNewValue();
     }
 
+    getPosition(selectedValue: string): number {
+        this.options.forEach((item, index) => {
+            if (selectedValue === item) {
+                if (this.selectedPosition < this.options.length) {
+                    this.selectedPosition = index;
+                }
+                return this.selectedPosition;
+            }
+        });
+        return this.selectedPosition;
```

Comment:
It can be simplified to 
`return this.options.findIndex(item => selectedValue === item)`